### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by harshsingh32**
This pr is under issue #19.

### CODE REVIEW

**Comment by AshutoshDash1999** [comment]
@harshsingh32 can you please add a logo of the GSSoC'23 batch to indicate that our project is part of GSSoC'23?

File: README.md [hunk 1]
```diff
@@ -40,10 +53,14 @@ b. To use the icons, click on your preferred icons, and paste inside your body t
...
-It's not possible to reach where we are now without our amazing contributors.
+We would like to express our heartfelt gratitude to the amazing contributors who have made this project possible. Their dedication and support have been invaluable in shaping and improving BookingJini Icons. We couldn't have come this far without them.
 
 <a href="https://github.com/Bookingjini-Labs/bookingjini-icons/graphs/contributors">
   <img src="https://contrib.rocks/image?repo=Bookingjini-Labs/bookingjini-icons" />
 </a>
+
+# Licence 🪪
+
+BookingJini Icons is released under the MIT License. You are free to use, modify, and distribute the icons in your projects, both personal and commercial.
```
Comment (AshutoshDash1999) [inline README.md]: Please remove the license part.

**Comment by AshutoshDash1999** [comment]
> @harshsingh32 can you please add a logo of the GSSoC'23 batch to indicate that our project is part of GSSoC'23?

@harshsingh32 this is on priority

**Comment by harshsingh32** [comment]
I would add the image.

**Comment by ramanujamgond** [comment]
@harshsingh32 - Kindly resolve the conflict.

**Pull request merged**